src/amo/components/CollectionAddAddon/styles.scss

```diff
@@ -2,21 +2,35 @@
 @import '~amo/css/inc/vars';
 
 .CollectionAddAddon {
+  margin-bottom: 24px;
+
   .ErrorList,
   .CollectionAddAddon-noticePlaceholder {
-    margin-top: 16px;
-    min-height: 32px;
+    margin-bottom: 6px;
   }
 
   .AutoSearchInput-label {
-    padding-top: 4px;
+    padding-bottom: 6px;
+  }
+
+  .AutoSearchInput-search-box {
+    padding-top: 6px;
   }
 
   .AutoSearchInput-query {
     border: 1px solid $grey-50;
     border-radius: $border-radius-xs;
   }
 
+  .AutoSearchInput-icon-magnifying-glass {
+    top: 12px;
+  }
+
+  .AutoSearchInput-submit-button,
+  .SearchSuggestion-icon-arrow {
+    display: none;
```

Comment:
This seems fine but I feel like their might be an accessibility issue with hiding the "button" (if anyone even knows that it's a button). We can worry about it later, though. The button was just a no-op before.